pulp_python/app/models.py

```diff
@@ -252,3 +254,10 @@ def finalize_new_version(self, new_version):
         """
         remove_duplicates(new_version)
         validate_repo_version(new_version)
+
+    @hook(BEFORE_UPDATE, when="remote", has_changed=True)
```

Comment:
You'll probably need to do the same thing when the `includes` or `url` has changed.